docs/backend-api/resources/tracking/tracker/index.md

```diff
@@ -460,24 +460,28 @@ List of state names for the field `states`:
 * **hardware_key** (driver identification key; value type: string).
 * **vibration_state** (value type: boolean).
 * **idling_state** (value type: boolean).
-* **external_power_state** (connected/disconnected; value type: boolean).
-* **case_intrusion_state** (value type: boolean).
-* **driver_ident_state** (identified/not identified; value type: boolean).
+* **external_power_state** (connected/disconnected; value type: string).
```

Comment:
Here we need a description of the possible values like you did it below. And what about `vibration_state`, `idling_state`? 
In fact, this is far from a complete list of possible state fields..

Reply:
Sure, I can add them all. Is there a way I can get information about all possible state fields and values? I added only for state fields I was confident the value is string and its meaning.

Reply:
added link to sensor/input_name/list API call with all descriptions.